### PR TITLE
Add hasDatabase function for checking DB exist

### PR DESCRIPTION
**Pull request by rrooij**

### CODE REVIEW

File: docs/api/woqlclient.md [hunk 1]
```diff
@@ -338,6 +338,27 @@ name the databases list will be set to empty
...
+
+
+| Param | Type | Description |
+| --- | --- | --- |
+| [orgName] | <code>string</code> | the organization id to set the context to |
+| [dbName] | <code>string</code> | the db name to set the context to |
+
+**Example**  
+```javascript
+async function callGetDatabases(){
```
Comment (GavinMendelGleason) [inline docs/api/woqlclient.md:356]: Is this the correct example?

**Review by GavinMendelGleason — CHANGES_REQUESTED** [review]

**Review by GavinMendelGleason — CHANGES_REQUESTED** [review]
comments do not match functions

**Comment by rrooij** [comment]
It does now right? Or am I missing something?

File: docs/api/woqlclient.md [hunk 2]
```diff
@@ -338,6 +338,29 @@ name the databases list will be set to empty
...
+
+
+| Param | Type | Description |
+| --- | --- | --- |
+| [orgName] | <code>string</code> | the organization id to set the context to |
+| [dbName] | <code>string</code> | the db name to set the context to |
+
+**Example**  
+```javascript
+async function executeIfDatabaseExists(f){
```
Comment (GavinMendelGleason) [inline docs/api/woqlclient.md:356]: What is the title of the doc string above?

**Review by GavinMendelGleason — APPROVED** [review]

File: docs/api/woqlclient.md (hunk 2 above)
Comment (rrooij) [reply docs/api/woqlclient.md:356]: hasDatabase. The `executeIfDatabaseExists` is just an example use case in which the function is called, not the function itself.

**Pull request merged**